assets/csidriveroperators/aws-ebs/07_clusterrole_aws_config.yaml

```diff
@@ -0,0 +1,15 @@
+# Allow reading openshift-config-managed/kube-cloud-config ConfigMap
+# with CA bundle.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Since the goal here is to allow reading a specific ConfigMap in a specific namespace, could we convert this `ClusterRole` into a `Role` and specify the namespace and the resource name? Like:
```
metadata:
  namespace: openshift-config-managed
(...)
  resources: 
  - configmaps
  resourceNames:
  - kube-cloud-config
  (...)
```

Reply:
I tried `resourceNames:`, but CSO failed to `watch` - it watches all configmaps in the namespace. I'll try `Role` referring to a ServiceAccount in a different namespace.

Reply:
Ok, it seems it does something - at least AWS EBS operator is not complaining about RBACs.